hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicRootedOzoneFileSystem.java

```diff
@@ -699,6 +709,8 @@ public boolean delete(Path f, boolean recursive) throws IOException {
   private boolean deleteInSpan(Path f, boolean recursive) throws IOException {
     incrementCounter(Statistic.INVOCATION_DELETE, 1);
     statistics.incrementWriteOps(1);
+    Span span = GlobalTracer.get().activeSpan();
+    span.setTag("path", f.toString());
```

Comment:
Here is also a tag setting the key instead of the path as its value. They should be set the same value for better trace. 
https://github.com/apache/ozone/blob/bab281cf0e3b23d4dd3520a06061f7e07ee96ca5/hadoop-ozone/ozonefs-common/src/main/java/org/apache/hadoop/fs/ozone/BasicRootedOzoneFileSystem.java#L248

Reply:
good catch. Updated the PR.